utils/poolNames.ts

```diff
@@ -75,6 +75,21 @@ export const getMarketInfoFromSymbol = (
     };
 };
 
+export const getSimplifiedTokenName = (tokenSymbol: string): string => {
+    // not 100% accurate but is unlikely other tokenSymbols have this form
+    const tokenSymbolRegex = /([0-9]*)(?:|L|S)\-([A-Z]*)/g;
+    const symbol = tokenSymbol.match(tokenSymbolRegex)?.[0];
```

Comment:
do we need to wrap `tokenSymbol` in `escapeRegex`?

Reply:
nah since tokenSymbol isnt the regex expression, tokenSymbolRegex is. TokenSymbol is jus the string, can be anything